todo_app/test/mongo_db_service_test.py

```diff
@@ -0,0 +1,66 @@
+import pytest
```

Comment:
Nice, a bonus set of tests for the MongoDBService - these definitely aren't trivial so nice work! 😄